src/components/StepsHeader.tsx

```diff
@@ -1,42 +1,47 @@
-import { Steps } from '../types'
-import { TransferType } from '@fiatconnect/fiatconnect-types'
+import { AppState, TransferInUserActionNoKycScreens, Screens } from '../types'
 
-interface Props {
-  step: Steps
-  transferType?: TransferType
+const SCREEN_NAME_MAP = {
+  [Screens.SignInScreen]: 'Sign In',
+  [Screens.PaymentInfoScreen]: 'Payment Info',
+  [Screens.KYCScreen]: 'KYC',
+  [Screens.ReviewScreen]: 'Review',
+  [Screens.UserActionDetailsScreen]: 'Pay',
+  [Screens.SendCryptoScreen]: 'Pay',
+  [Screens.DoneScreen]: 'Done', // This will never be shown
 }
 
-const MAX_STEPS = 4
-
-const STEP_TO_TITLE: Record<number, string> = {
-  1: 'Sign In',
-  2: 'Payment Info',
-  3: 'Review',
-  4: 'Pay',
+interface Props {
+  appState?: AppState
 }
 
-export function StepsHeader({
-  step,
-  transferType = TransferType.TransferIn,
-}: Props) {
-  // TODO M2: Update this to take into account transfers out
+const DEFAULT_SCREENS = TransferInUserActionNoKycScreens
```

Comment:
might be better to require kyc by default, to avoid an unpleasant surprise if something loads and a new step is added that wasn't there before or something. I haven't fully grokked how this value is used yet though